support/crown/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "crown"
-version.workspace = true
-authors.workspace = true
-license.workspace = true
-edition.workspace = true
-publish.workspace = true
+authors = ["The Servo Project Developers"]
+version = "0.0.1"
```

Comment:
I think in general it makes sense to version `crown` independently, so that in the future `./mach bootstrap` or perhaps even `build` could detect if crown needs to be updated.